package.json

```diff
@@ -34,6 +34,7 @@
   "scripts": {
     "generateParser": "cd src/parser; rm -rf ./generated/*; java -Xmx500M -cp \"../../antlr-4.7.1-complete.jar:$CLASSPATH\" org.antlr.v4.Tool -o generated -Dlanguage=JavaScript FHIRPath.g4",
     "build:browser": "cp LICENSE.md browser-build && cd browser-build && webpack && rm -f fhirpath.zip && zip fhirpath.zip LICENSE.md fhirpath.min.js && rm LICENSE.md",
+    "build:browser-package": "npm run build && cp ./browser-build/fhirpath.min.js ./browser-package/fhirpath.min.js",
```

Comment:
I'm not sure what you are trying to achieve here.  It looks like you have just copied fhirpath.min.js.  In your comment, you had a .tgz file highlighted.  Are you trying to replace the .zip with .tgz?

Reply:
The highlighting was just the last thing I clicked on. The tgz is the result of npm pack and should only be used as an npm package.

fhirpath.min.js is copied because that has (as far as I can tell) the functionality the application needs without the rest of the repo being copied in as was happening previously.

The zip file and its purpose is not modified by this PR.

Reply:
browser-build/fhirpath.min.js is what you would use if you wanted to include fhirpath.js in your page via a script tag, but in that case I think there would not be any reason to install the package via npm-- you could use one of the published releases (downloading the fhirpath.zip file from the releases page).

The reason to install via npm is if you are gong to bundle up fhirpath.js along with your application files and other dependencies.

Are you trying to reduce the size of node_modules/fhirpath?  I see we do include several folders there (browser-build being one, but also demo and test) which are not really needed to use the package.  We could exclude those from the npm package, and perhaps should, though the total is currently under 1 MB.

Reply:
This reduces the size and removes the dependency on "node": ">=8.9.0"

Reply:
The size of what?

Reply:
Okay, I think I understand that your are trying to reduce the size of node_modules/fhirpath that results from npm install fhirpath.  (Please confirm.)

I don't think that creating a separate directory with its own package.json is the right approach to do that.  We can unnecessary files with .npmignore.  Also, browser-build/fhirpath.min.js is not the right file for npm.  It is meant for browsers, and requiring it will put "fhirpath" variable into the global scope.

The other concern you mentioned was about removing the requirement of  "node": ">=8.9.0".  Since the oldest supported Node.js release is 8, why would you want to remove that?  Are you running an unsupported version of Node.js?